README.md

```diff
@@ -46,30 +46,38 @@ Wouter provides a simple API that many developers and library authors appreciate
 
 ## Table of Contents
 
+- [Features](#features)
+- [developers :sparkling_heart: wouter](#developers-sparkling_heart-wouter)
+- [Table of Contents](#table-of-contents)
 - [Getting Started](#getting-started)
-- [API](#wouter-api)
-  - **[Hooks](#hooks-api)**
-    - **[`useRoute`](#useroute-the-power-of-hooks)**
-    - **[`useLocation`](#uselocation-hook-working-with-the-history)**
-    - **[`useRouter`](#userouter-accessing-the-router-object)**
-  - **[Components](#component-api)**
-    - **[`<Route />`](#route-pathpattern-)**
-    - **[`<Link />`](#link-hrefpath-)**
-    - **[`<Switch />`](#switch-)**
-    - **[`<Redirect />`](#redirect-topath-)**
-    - **[`<Router />`](#router-hookhook-matchermatchfn-basebasepath-)**
-      - [Matching Dynamic Segments](#matching-dynamic-segments)
-      - [Using a `path-to-regexp`-based matcher](#using-a-path-to-regexp-based-matcher)
-- [FAQ and How-to's](#faq-and-code-recipes)
-  - [Base path](#i-deploy-my-app-to-the-subfolder-can-i-specify-a-base-path)
-  - [Default route](#how-do-i-make-a-default-route)
-  - [Active links](#how-do-i-make-a-link-active-for-the-current-route)
-  - [Nested routes](#are-relative-routes-and-links-supported)
-  - [Multipath routes](#is-it-possible-to-match-an-array-of-paths)
-  - [TypeScript support](#can-i-use-wouter-in-my-typescript-project)
-  - [Using with Preact](#preact-support)
-  - [Server-side Rendering (SSR)](#is-there-any-support-for-server-side-rendering-ssr)
-  - [Routing in less than 400B](#1kb-is-too-much-i-cant-afford-it)
+  - [Supporting IE11 and obsolete platforms](#supporting-ie11-and-obsolete-platforms)
```

Comment:
Good idea, let's keep that one in the toc!
```suggestion
  - [Browser Support](#supporting-ie11-and-obsolete-platforms)
```

Reply:
👍